src/coreclr/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -471,6 +475,11 @@ bool CommandLine::Parse(int argc, char* argv[], /* OUT */ Options* o)
                     }
                 }
             }
+            else if ((_strnicmp(&argv[i][1], "failureLimit", argLen) == 0))
+            {
+                ++i;
+                o->failureLimit = atoi(argv[i]);
```

Comment:
You need to check that someone actually provided an argument, e.g.,
```
                if (i >= argc)
                {
                    DumpHelp(argv[0]);
                    return false;
                }
```

Reply:
Seems like you should also ensure failureLimit >= 1

Reply:
ah, my bad.